server/data/enums/role.ts

```diff
@@ -11,9 +11,10 @@ export enum Role {
   DeleteSensitiveCaseNotes = 'ROLE_DELETE_SENSITIVE_CASE_NOTES',
   DietAndAllergiesEdit = 'ROLE_DIET_AND_FOOD_ALLERGIES_EDIT',
   DpsApplicationDeveloper = 'ROLE_DPS_APPLICATION_DEVELOPER',
-  EditEducationWorkPlan = 'ROLE_EDUCATION_WORK_PLAN_EDITOR',
   GlobalSearch = 'ROLE_GLOBAL_SEARCH',
   InactiveBookings = 'ROLE_INACTIVE_BOOKINGS',
+  LearningAndWorkProgressContributor = 'ROLE_LWP_CONTRIBUTOR',
```

Comment:
The LWP Contributor role is not used within DPS Prisoner Profile, but I've defined it here for completeness